client/lib/plans/constants.js

```diff
@@ -1119,7 +1119,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_FREE_BLOG_DOMAIN ]: {
 		getSlug: () => FEATURE_ADVANCED_CUSTOMIZATION,
-		getTitle: () => i18n.translate( 'Free .blog domain for one year' ),
+		getTitle: () => i18n.translate( 'Free .blog Domain for One Year' ),
```

Comment:
I'm not certain where or even whether this string is being used, but there are several features whose titles are not capitalized. The capitalized ones are used in the plans pages (both /plans and signup). I've left the capitalization in place and just updated the content for policy change.